FormData.js

```diff
@@ -124,7 +127,7 @@ class FormDataPolyfill {
    * @return  {Array}   [name, value]
    */
   getAll(name) {
-    return (wm(this)[name += ''] || []).concat()
+    return (wm(this)[name += ''] || []).concat().map(v => v[0])
```

Comment:
You can remove `.concat()`, Since I used it to get a copy of the array (which the map value now already dose)